lib/tasks/export_local_transactions.rake

```diff
@@ -0,0 +1,15 @@
+desc "Export Local Transactions' slugs, names, LGSL and optional LGIL codes as CSV"
+
+task :export_local_transactions => :environment do
+  require "csv"
+
+  csv_string = CSV.generate do |csv|
+    csv << ["slug","lgsl","lgil","title"]
+
+    Edition.where(_type: "LocalTransactionEdition").each do |lte|
+      csv << [lte.slug,lte.lgsl_code,lte.lgil_override,lte.title]
```

Comment:
Is that anything to worry about? I was surprised everything didn't have a `lgil_code`, and settled for the sometimes present `lgil_override` as per the Publishing admin interface fields.